contracts/eden/src/inductions.cpp

```diff
@@ -169,17 +167,15 @@ namespace eden
          endorsement_idx.modify(itr, eosio::same_payer, [](auto& row) { row.endorsed() = true; });
          itr++;
       }
-      maybe_create_nft(induction);
```

Comment:
since you removed the `maybe_create_nft` from this function and we won't charge donation for genesis, you need to go inside the `inductprofil` and call `create_nft` after `endorse_all`. 

Now that I'm thinking about this function, I think we can rename this to `complete_genesis_induction` (since it's only used for this anyway) and instead of removing the `maybe_create_nft` just call `create_nft`.

Reply:
I thought we were going to charge initiation fees for genesis. @brandonfancher? @cool-ant?